packages/node/src/index.ts

```diff
@@ -41,4 +44,40 @@ export default class LogtoClient extends BaseClient {
       generateState,
     });
   }
+
+  getContext = async (getAccessToken = false) => {
+    const { isAuthenticated } = this;
+
+    if (!isAuthenticated) {
+      const user: LogtoContext = {
+        isAuthenticated,
+      };
+
+      return user;
```

Comment:
nit: Return directly without declaring the `user` variable?

Reply:
updated